src/org/thoughtcrime/securesms/database/SmsDatabase.java

```diff
@@ -81,7 +81,7 @@
     DELIVERY_RECEIPT_COUNT + " INTEGER DEFAULT 0," + SUBJECT + " TEXT, " + BODY + " TEXT, " +
     MISMATCHED_IDENTITIES + " TEXT DEFAULT NULL, " + SERVICE_CENTER + " TEXT, " + SUBSCRIPTION_ID + " INTEGER DEFAULT -1, " +
     EXPIRES_IN + " INTEGER DEFAULT 0, " + EXPIRE_STARTED + " INTEGER DEFAULT 0, " + NOTIFIED + " DEFAULT 0, " +
-    READ_RECEIPT_COUNT + " INTEGER DEFAULT 0);";
+    READ_RECEIPT_COUNT + " INTEGER DEFAULT 0," + MARKED_UNREAD +" INTEGER DEFAULT 0);";
```

Comment:
Not something that really has to be changed, but doesn't the app already have a way of showing the users that something is unread? Why do we need to add more things ui-based if we can simply toggle what it already does?

Reply:
I could possibly rename this to readReminder to avoid confusion?

Reply:
Yes, I think that would work. Wrong comment though.